.env.example

```diff
@@ -23,7 +23,7 @@ GITHUB_TOKEN={fill me in}
 # BYPASS_DETAILS={optional 'Some text explaining bypass procedure'}
 # BUDDY_CHECK_FEATURE={optional, set to 1 to enable buddy_check feature; 0 to disable}
 
-# DEPLOY_MAX_MINUTES_PENDING={optional, set to 20 for max minutes a deploy is pending}
+# BUDDY_CHECK_TIME_LIMIT={optional, set to 20 for max minutes a deploy is pending}
```

Comment:
ahhh the other value is BUDDY_CHECK_GRACE_PERIOD ... so no need for any
changes :D

On Thu, Jul 2, 2015 at 1:36 PM, Roman Sandler <notifications@github.com>
wrote:

> In .env.example
> <https://github.com/zendesk/samson/pull/502#discussion_r33820034>:
>
> > @@ -23,7 +23,7 @@ GITHUB_TOKEN={fill me in}
> >  # BYPASS_DETAILS={optional 'Some text explaining bypass procedure'}
> >  # BUDDY_CHECK_FEATURE={optional, set to 1 to enable buddy_check feature; 0 to disable}
> >
> > -# DEPLOY_MAX_MINUTES_PENDING={optional, set to 20 for max minutes a deploy is pending}
> > +# BUDDY_CHECK_TIME_LIMIT={optional, set to 20 for max minutes a deploy is pending}
>
> so there are two different scenarios:
> 1. I deploy production, but within BUDDY_CHECK_TIMEOUT nobody approves my
> deploy. It gets cancelled.
> 2. I deploy production:datacenter1 with a buddy and some time later I try
> to deploy the same tag to production:datacenter2. If BUDDY_CHECK_TIMEOUT2
> has elapsed then I need another approval.
>
> Do we need separate variables here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/502/files#r33820034>.
>